styles/google-gemini/catppuccin.user.css

```diff
@@ -232,9 +239,19 @@
 
     --og-theme-color: @text;
 
+    span {
+        color: @text
+    }
```

Comment:
What is this for? if you are trying to style "From your ip address" text you should use `--gem-sys-color--primary: @accent-color;

also you missing semicolon

Reply:
this is for the other text that can't be themed using vars

Reply:
This overrides some of variables

can you tell me which text cant be styled? i cant find any

Reply:
"New chat" button and the location at the bottom

Reply:
New chat button, can be styled using
```
--bard-color-on-surface-rgb: #rgbify(@text) [];
--gem-sys-color--on-surface--rgb: #rgbify(@text) [];
--gem-sys-color--surface-dim--rgb: #rgbify(@crust) [];
```

Location seems to be styled, not sure what you meant

Reply:
> New chat button can be styled using
> 
> ```
> --bard-color-on-surface-rgb: #rgbify(@text) [];
> --gem-sys-color--on-surface--rgb: #rgbify(@text) [];
> --gem-sys-color--surface-dim--rgb: #rgbify(@crust) [];
> ```
> 
> Location seems to be styled, not sure what you meant

i guess i'll remove span and add these variables